appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java

```diff
@@ -95,8 +95,8 @@ private Index getIndex() {
     } catch (SearchException e) {
       // handle exception...
```

Comment:
For the record, we really shouldn't be doing this.  We should actually handle the exception or propagate it.